### ISSUE TITLE
NTR: Anti-RNA-polymerase-III-autoantibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5799]
**Preferred term label:**
Anti-RNA-polymerase-III-autoantibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react to RNA-polymerase III.

Comment: Anti-RNA-polymerase III- autoantibodies can be detected in patients with systemic sclerosis.

	PMID: 31858335
  	DOI: 10.1007/s10067-019-04806-9

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:90291   (Disorder)   Systemic sclerosis

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese
